backend/src/layer/logic/mealplan_management/test/mealplan_management_database_mock.rs

```diff
@@ -0,0 +1,241 @@
+//! This crate contains mocks of [`MealplanManagementDatabaseMock`] for testing.
+
+use async_trait::async_trait;
+
+use crate::{
+    interface::persistent_data::{
+        model::{Canteen, Line, Meal, Side},
+        MealplanManagementDataAccess, Result,
+    },
+    util::{
+        Additive, Allergen, Date,
+        MealType::{self, Vegan},
+        Price, Uuid,
+    },
+};
+
+const fn get_price() -> Price {
+    Price {
+        price_student: 0,
+        price_employee: 0,
+        price_guest: 0,
+        price_pupil: 0,
+    }
+}
+
+const fn get_price_by_reference(price: &Price) -> Price {
```

Comment:
did you just implement the clone trait 🤣 ?

Reply:
This is actually funny and hilarious...

Reply:
If they dont take too long to complete (and they will run in parallel), we can just let them in there..